vendor/config.py

```diff
@@ -19,7 +19,10 @@ class PaymentProcessorForm(forms.Form):
 
 
 class PaymentProcessorSiteSelectForm(PaymentProcessorForm):
-    site = forms.CharField(label=_("Site"), widget=forms.Select(choices=[(site.pk, site.domain) for site in Site.objects.all()]))
+    site = forms.CharField(label=_("Site"))
+
+    def __init__(self, *args, **kwargs):
+        self.fields['site'].widget = forms.Select(choices=[(site.pk, site.domain) for site in Site.objects.all()])
```

Comment:
not a CR- is there a reason this had to move to the init from the field declaration?

Reply:
Yes, django was having conflicts with the db. It was trying to get the django_site table before it was created. So it would through an error. You could not even run migrations because of it when it was in the field declaration or even in the Meta class.

Reply:
I actually changed that in django-vendor-promo also